lib/support/insertNode.js

```diff
@@ -1,17 +1,19 @@
+/* eslint-disable camelcase */
 module.exports = function insertNode(node, parent, key, version, optimizedPath) {
-    // eslint-disable-next-line camelcase
     node.$_key = key;
-    // eslint-disable-next-line camelcase
     node.$_parent = parent;
 
     if (version !== undefined) {
-        // eslint-disable-next-line camelcase
         node.$_version = version;
     }
-    // eslint-disable-next-line camelcase
     if (!node.$_absolutePath) {
-        // eslint-disable-next-line camelcase
-        node.$_absolutePath = optimizedPath.slice(0, optimizedPath.index).concat(key);
+        if (Array.isArray(key)) {
```

Comment:
The original code was apparently calling `concat` with a single element as the argument, TIL you can do that.

I'm not sure if `key` is ever an array, added this block for parity just in case.